docs/v3/deploy/infrastructure-examples/managed.mdx

```diff
@@ -95,19 +106,19 @@ Remote block-based storage is also supported, so S3, GCS, and Azure Blob are add
 
 ### Resources
 
-Memory is limited to 2 GB of RAM, which includes all operations such as dependency installation. 
+Memory is limited to 2 GB of RAM, which includes all operations such as dependency installation.
 Maximum job run time is 24 hours.
 
 ## Usage limits
 
-Free tier accounts are limited to 10 compute hours per workspace per month. 
-Pro tier and above accounts are limited to 250 hours per workspace per month. 
+Free tier accounts are limited to 10 compute hours per workspace per month.
+Pro tier and above accounts are limited to 250 hours per workspace per month.
 You can view your compute hours quota usage on the **Work Pools** page in the UI.
 
 ## Next steps
 
 Read more about creating deployments in [Run flows in Docker containers](/v3/deploy/infrastructure-examples/docker/).
 
-For more control over your infrastructure, such as the ability to run 
-custom Docker images, [serverless push work pools](/v3/deploy/infrastructure-examples/serverless/) 
+For more control over your infrastructure, such as the ability to run
+custom Docker images, [serverless push work pools](/v3/deploy/infrastructure-examples/serverless/)
 are also a good option.
```

Comment:
not your diff but i feel like 'also' implies Managed gives control, which is not what we mean to say
```suggestion
are a good option.
```

Reply:
good catch. words are hard